DuckDuckGoTests/StatisticsLoaderTests.swift

```diff
@@ -270,6 +276,19 @@ class StatisticsLoaderTests: XCTestCase {
         waitForExpectations(timeout: 5, handler: nil)
     }
 
+    func testWhenInstallStatisticsRequestedThenInstallPixelIsFired() {
+        loadSuccessfulExiStub()
+
+        let testExpectation = expectation(description: "refresh complete")
+        testee.refreshAppRetentionAtb {
+            Thread.sleep(forTimeInterval: .seconds(0.1))
+            testExpectation.fulfill()
```

Comment:
This small delay isn't ideal; it's in place since the `refreshAppRetentionAtb` callback is waiting for the `exti` call, **not** the pixel call, so there is a chance that this could complete before the pixel call has had time to do its thing.

It would be better to wait for the pixel firing mock to have the right value, with a ~1 second timeout. I'll look at updating this.